taxonomies/ingredients_processing.txt

```diff
@@ -361,7 +361,7 @@ en:zest
 
 # en:description:The soft center of a fruit
 en:pulp
-de:fruchtfleisch
+de:mark, pulpe, fruchtfleisch
```

Comment:
"Passierte Fruchtmassen werden ebenfalls als Mark bezeichnet, zum Beispiel [Tomatenmark](https://de.wikipedia.org/wiki/Tomatenmark) oder Himbeermark. Fruchtmark kann Säften oder Nektaren und vielen weiteren Nahrungsmitteln zugesetzt werden." [Wikipedia](https://de.wikipedia.org/wiki/Mark_(Lebensmittel))
So "mark" is also used for other things and I even think it is used more often in these other ways.
According to that evidence do we have to change sth?